lambdas/get-addresses/tests/unit/app.test.ts

```diff
@@ -1,26 +1,19 @@
-jest.mock("../../src/services/config-service", () => {
-    return {
-        ConfigService: jest.fn().mockImplementation(() => {
-            return {
-                init: jest.fn().mockResolvedValue([]),
-                config: {
-                    AddressLookupTableName: "Test",
-                },
-            };
-        }),
-    };
-});
-
+jest.mock("@aws-lambda-powertools/parameters/ssm", () => ({
+    getParameter: jest.fn(),
+}));
+import { getParameter } from "@aws-lambda-powertools/parameters/ssm";
 import { APIGatewayProxyEvent } from "aws-lambda";
 import { lambdaHandler } from "../../src/app";
 import { CanonicalAddress } from "../../src/types/address";
 import { DynamoDbClient } from "../../src/lib/dynamo-db-client";
 
 const mockDynamoDbClient = jest.mocked(DynamoDbClient);
+const mockedGetParameter = getParameter as jest.MockedFunction<typeof getParameter>;
```

Comment:
```suggestion
const dynamoDbClientMock = jest.mocked(DynamoDbClient);
const getParameterMock = getParameter as jest.MockedFunction<typeof getParameter>;
```
Both should be either `mocked...` or `mock...` for consitency - or better `...Mock`